src/dawn/engine_dawn.cc

```diff
@@ -1169,12 +1169,12 @@ Result EngineDawn::DoCompute(const ComputeCommand*) {
 }
 
 Result EngineDawn::DoEntryPoint(const EntryPointCommand*) {
-  return Result("Dawn:DoEntryPoint not implemented");
+  return Result("Dawn: Entry point must be \"main\" in Dawn");
```

Comment:
I think this is true now but will be relaxed in the future.
I'll file an issue to enhance this later.